src/coprocessor/dag/batch/executors/util/scan_executor.rs

```diff
@@ -211,10 +211,11 @@ impl<S: Store, I: ScanExecutorImpl, P: PointRangePolicy> BatchExecutor for ScanE
         assert!(!self.is_ended);
         assert!(scan_rows > 0);
 
-        let mut data = self.imp.build_column_vec(scan_rows);
-        let is_drained = self.fill_column_vec(scan_rows, &mut data);
+        let mut logical_columns = self.imp.build_column_vec(scan_rows);
```

Comment:
Why is this called `logical_columns`?

Reply:
All data accessed are logical data, for all kind of executors. It is only that in scan executor physical columns equal to logical columns. So it is still called logical columns, but assigned to physical columns field with a logical_rows equal to 0..N.